<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DFits</class>
 <widget class="QMainWindow" name="DFits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DFits</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="HDU_List">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="mini_light_curve_plot" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tab_widget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MyStatusBar" name="statusbar" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="actionClose_File"/>
    <addaction name="actionxport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuImage">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Image</string>
     </property>
     <addaction name="actionColormap"/>
    </widget>
    <widget class="QMenu" name="menuStatistics">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Statistics</string>
     </property>
     <addaction name="actionLight_Curve"/>
    </widget>
    <widget class="QMenu" name="menuMarker">
     <property name="title">
      <string>Marker</string>
     </property>
     <addaction name="actionList_Markers"/>
     <addaction name="actionDelete_Markers"/>
    </widget>
    <addaction name="menuImage"/>
    <addaction name="menuStatistics"/>
    <addaction name="menuMarker"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionFit_to_Width"/>
    <addaction name="actionFit_to_Height"/>
    <addaction name="actionZoom0"/>
    <addaction name="actionZoom25"/>
    <addaction name="actionZoom50"/>
    <addaction name="actionZoom75"/>
    <addaction name="actionZoom100"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuZoom"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen_toolbar"/>
   <addaction name="separator"/>
   <addaction name="actionoverview"/>
   <addaction name="actionoverview_raw"/>
   <addaction name="actionSave_toolbar"/>
   <addaction name="action_export_toolbar"/>
   <addaction name="actionDeleteAllMarkers"/>
   <addaction name="actionMarkerMode"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionColormap">
   <property name="text">
    <string>Colormap</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionoverview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/file-color-icon.svg</normaloff>../../Downloads/file-color-icon.svg</iconset>
   </property>
   <property name="text">
    <string>overview</string>
   </property>
   <property name="toolTip">
    <string>Overview</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionopen_toolbar">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/file-manager-icon.svg</normaloff>../../Downloads/file-manager-icon.svg</iconset>
   </property>
   <property name="text">
    <string>open_toolbar</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionoverview_raw">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="applications-development"/>
   </property>
   <property name="text">
    <string>Raw Overview</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLight_Curve">
   <property name="text">
    <string>Light Curve</string>
   </property>
  </action>
  <action name="actionxport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export As</string>
   </property>
  </action>
  <action name="actionSave_toolbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/diskette-save-svgrepo-com.svg</normaloff>../../Downloads/diskette-save-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_export_toolbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/export-svgrepo-com.svg</normaloff>../../Downloads/export-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>export</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDeleteAllMarkers">
   <property name="text">
    <string>DeleteAllMarkers</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFit_to_Width">
   <property name="text">
    <string>Fit to Width</string>
   </property>
  </action>
  <action name="actionFit_to_Height">
   <property name="text">
    <string>Fit to Height</string>
   </property>
  </action>
  <action name="actionZoom0">
   <property name="text">
    <string>0%</string>
   </property>
  </action>
  <action name="actionZoom25">
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="actionZoom50">
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="actionZoom75">
   <property name="text">
    <string>75%</string>
   </property>
  </action>
  <action name="actionZoom100">
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="actionMarkerMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="applications-graphics"/>
   </property>
   <property name="text">
    <string>MarkerMode</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionList_Markers">
   <property name="text">
    <string>List Markers</string>
   </property>
  </action>
  <action name="actionDelete_Markers">
   <property name="text">
    <string>Delete Markers</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyStatusBar</class>
   <extends>QWidget</extends>
   <header location="global">mystatusbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DFits</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
